docs/build/marmalade-tutorial.md

```diff
@@ -0,0 +1,398 @@
+
+This is due to policies’ customizable nature, which allows them to enforce different parameters. For example, one policy might need an amount for royalty whereas another might need a specific keyset to allow token purchases.
+
+Here is an example of the creation of a fixed quote policy token:
+
+```
+{
+    "hash": "0wNWu4Of36FGGOu5PznASvULdmjn4vsjmaoMGeUl-BM",
+    "cmd": "{
+"networkId":"testnet04",
+"signers":[{"clist":[{"name":"coin.GAS","args":[]}],"pubKey":"e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"}],
+"meta":{"creationTime":1655843658,"ttl":28800,"gasLimit":100000,"chainId":"1","gasPrice":0.000001,"sender":"mike.tanto"},
+"nonce":""2022-06-21T20:34:33.312Z"",
+//This is where to pass in "exec" environment data
+//As seen for fixed quote, we need to pass in "mint-guard" "max-supply" "min-amount"
+
+
+"Payload":{"exec":{"data":
+{
+"manifest": "some-manifest"
+"mint-guard":{"pred":"keys-all","keys":["e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"]},
+"Max-supply":"100.0",
+"Min-amount":"0.0"
+},
+
+//This is the pact function call code; 
+//read-msg finds the data with appropriate variable name from environment data
+"code":"(marmalade.ledger.create-token "MKOCOIN" 12 (read-msg 'manifest) marmalade.fixed-quote-policy)"}}}",
+    "sigs": {
+        "e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2": null
+    } 
+}
+```
+
+Here is an example of the creation of a guard policy token:
+
+```
+{"hash":"-NAUtAL0-xAssDrve6V-dddOUlY-88FBGxlGJHMgid8",
+"cmd":"{"networkId":"testnet04","signers":[{"clist":[{"name":"coin.GAS","args":[]}],
+"pubKey":"e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"}],
+"meta":{"creationTime":1655845010,"ttl":28800,"gasLimit":100000,"chainId":"1","gasPrice":0.000001,"sender":"mike.tanto"},"nonce":""2022-06-21T20:57:05.092Z"",
+//This is where to pass in "exec" environment data
+//As seen for fixed quote, we need to pass in "mint-guard" "max-supply" "min-amount
+"Payload":{"exec":{"data":{ "manifest” : ”some-manifest”, "mint-guard":{"pred":"keys-all","keys":["e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"]},
+"Burn-guard":{"pred":"keys-all","keys":["e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"]},
+"Sale-guard":{"pred":"keys-all","keys":["e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"]},
+"Transfer-guard":{"pred":"keys-all","keys":["e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2"]}},
+"code":"(marmalade.ledger.create-token "MKOCOIN" 12 (read-msg 'manifest) marmalade.guard-token-policy)"}}}",
+"sigs":{"e6ee763bd659fb2bb4e1f402f338e4bb374b91434f34d5fd49d85e99a00df9e2":null}}
+``` 
+
+**Minting a Token**: Minting is done through the `mint` function, which takes parameters `id` (token name), `account` (minter account name), `guard` (keyset of minter), and `amount` (decimal amount to mint). 
+
+The function call looks as simple as this:
+
+```
```

Comment:
to have syntax highlighting in a code block use \`\`\`clojure for pact and \`\`\`js for javascript
https://docusaurus.io/docs/markdown-features/code-blocks#syntax-highlighting